### DISCUSSION TITLE
issue with char in golang .go file

### DISCUSSION THREAD

**Discussion by kaykhan** [lukas-reineke/indent-blankline.nvim#897] [General]
I seem to be having an issue with golang project - the indent char  seem to become an up arrow

![image](https://github.com/lukas-reineke/indent-blankline.nvim/assets/5202527/d3d234de-e898-4096-8285-ed67a986948f)

```
local highlight = {
	"RainbowRed",
	"RainbowYellow",
	"RainbowBlue",
	"RainbowOrange",
	"RainbowGreen",
	"RainbowViolet",
	"RainbowCyan",
}

local hooks = require("ibl.hooks")
-- create the highlight groups in the highlight setup hook, so they are reset
-- every time the colorscheme changes
hooks.register(hooks.type.HIGHLIGHT_SETUP, function()
	vim.api.nvim_set_hl(0, "RainbowRed", { fg = "#E06C75" })
	vim.api.nvim_set_hl(0, "RainbowYellow", { fg = "#E5C07B" })
	vim.api.nvim_set_hl(0, "RainbowBlue", { fg = "#61AFEF" })
	vim.api.nvim_set_hl(0, "RainbowOrange", { fg = "#D19A66" })
	vim.api.nvim_set_hl(0, "RainbowGreen", { fg = "#98C379" })
	vim.api.nvim_set_hl(0, "RainbowViolet", { fg = "#C678DD" })
	vim.api.nvim_set_hl(0, "RainbowCyan", { fg = "#56B6C2" })
end)

require("ibl").setup({ indent = { highlight = highlight, char = "▏" } })
```

in typescript it is fine

![image](https://github.com/lukas-reineke/indent-blankline.nvim/assets/5202527/e8011fae-fce8-4753-afd1-7fbc3f918f51)

**Comment by lukas-reineke** [comment]
You have `:help 'list'` set, but no tab character configured under `:help 'listchars'`.

**Reply by kaykhan** [reply]
> You have `:help 'list'` set, but no tab character configured under `:help 'listchars'`.

Right, so i added 

vim.opt.listchars = {
	space = "·",
	eol = "↴",
	tab = "» ",
	trail = "·",
	nbsp = "␣",
}

but the tab icon seems to overwrite the ibl char

**Reply by lukas-reineke** [reply]
> the tab icon seems to overwrite the ibl char

Yes, that's the correct behavior. Please read `:help ibl.config.indent.tab_char`
